src/components/molecules/achievement/index.ts

```diff
@@ -0,0 +1 @@
+export { Achievement } from './achievement';
```

Comment:
```suggestion
export { Achievement } from './achievement';
export type { AchievementProps } from './types';
```

Пропы тоже экспортируем

Reply:
Насколько я вижу, комментарий не исправлен.